.github/workflows/build.yml

```
@@ -0,0 +1,15 @@
+name: build
+
+on:
+  release:
+    types: [ released ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    with:
+      upload_to_pypi: ${{ github.event_name == 'released'}}
```

```suggestion
      upload_to_pypi: ${{ (github.event_name == 'release') && (github.event.action == 'released') }}
```

@braingram is correct, however we should only be publishing full releases to PyPi, not just anything which is a release even such as editing an existing release's description. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release.